src/main/java/me/itzg/helpers/modrinth/ModrinthCommand.java

```diff
@@ -236,8 +247,14 @@ private Stream<? extends Path> processProject(String projectRef) {
 
```

Comment:
What you have is really good.
To keep the version splitting local to this method, perhaps you could do the split here with limit 2.